sdks/python/apache_beam/io/textio.py

```diff
@@ -484,6 +485,9 @@ def __init__(
         from each source file. Must be 0 or higher. Large number of skipped
         lines might impact performance.
       coder: Coder used to decode each line.
+      with_context: If True, returns a Key Value with the key being the file
```

Comment:
Not sure if you noticed, but we also have a "ReadFromTextWithFilename". Probably good to describe the difference between the two in the module level documentation.

Reply:
I added a clarification on the doc, not sure if it's good enough